src/lib/openapi/spec/role-schema.ts

```diff
@@ -3,20 +3,32 @@ import { FromSchema } from 'json-schema-to-ts';
 export const roleSchema = {
     $id: '#/components/schemas/roleSchema',
     type: 'object',
+    description:
+        'A role holds permissions to allow Unleash to decide what actions a roleholder is allowed to perform',
```

Comment:
Using 'account holder' as reference, I suggest splitting 'role holder' into two words.

```suggestion
        'A role holds permissions to allow Unleash to decide what actions a role holder is allowed to perform',
```